### PR TITLE
Balance

### PR DESCRIPTION
**Pull request by caesar003**

### CODE REVIEW

**Comment by caesar003** [comment]
run database migration,

```sh
php artisan migrate
```

- now enroll any course as a student, and it should now inserted into `transactions` table instead of `payments`. 
- log out, and log back in as the teacher, under **Teacher dashboard** > **Balance** now there is this information abour transaction history regarding this particular user,
- the withdraw form is working, put any number and submit it, it should be added into the table with balance value adjusted

File: app/Http/Controllers/AdminController.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+{
+    public function index()
+    {
+        if (auth()->user()->role !== 'admin') {
+            return redirect('/');
+        }
+
+        return redirect(route('admin.users'));
+
+        // return view('admin.index');
```
Comment (ahmadrosid) [inline app/Http/Controllers/AdminController.php:19]: remove this debug code

File: app/Http/Controllers/CourseController.php [hunk 2]
```diff
@@ -14,6 +14,9 @@ class CourseController extends Controller
 {
     public function index(Request $request)
     {
+        if (auth()->user() && auth()->user()->role === 'admin') {
+            return redirect('/admin/');
+        }
```
Comment (ahmadrosid) [inline app/Http/Controllers/CourseController.php:19]: why we redirect admin from here?

File: create-admin.sh [hunk 3]
```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```
Comment (ahmadrosid) [inline create-admin.sh:1]: can we do this using php artisan instead?

File: resources/views/admin/index.blade.php [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<x-admin-layout>
+    <div class="p-5"></div>
+</x-admin-layout>
```
Comment (ahmadrosid) [inline resources/views/admin/index.blade.php:3]: what is this for?

File: create-admin.sh (hunk 3 above)
Comment (caesar003) [reply create-admin.sh:1]: I'll figure it out sir,

File: app/Http/Controllers/CourseController.php (hunk 2 above)
Comment (caesar003) [reply app/Http/Controllers/CourseController.php:19]: I suppose, there's no need to show the dashboard for admin.

File: create-admin.sh (hunk 3 above)
Comment (ahmadrosid) [reply create-admin.sh:1]: You can add menu on sidebar for that

**Comment by caesar003** [comment]
to create an user with admin role:

```sh
php artisan app:create-admin
```

**Review by ahmadrosid — APPROVED** [review]

**Pull request merged**